pkgs/by-name/pa/pa-notify/package.nix

```
@@ -0,0 +1,39 @@
+{ lib
+, pkgs
+, stdenv
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "pa-notify";
+  version = "1.5.0";
+
+  src = fetchFromGitHub {
+    owner = "ikrivosheev";
+    repo = "pa-notify";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-356qwSxxxAUNJajsVjH3zqGAZQwMOcoLPSKPZdsCmBM=";
+  };
+
+  nativeBuildInputs = [
+    pkgs.cmake
+  ];
+
+  buildInputs = with pkgs; [
+    extra-cmake-modules
+    glib.dev
+    libnotify
+    libpulseaudio
+    pkg-config
+    util-linux.dev
+  ];
+
+  meta = {
+    homepage = "https://github.com/ikrivosheev/pa-notify";
+    description = "PulseAudio or PipeWire volume notification";
+    license = with lib.licenses; [ mit ];
+    maintainers = with lib.maintainers; [ juancmuller ];
+    mainProgram = "pa-notify";
+    platforms = lib.platforms.unix;
+  };
```

`with lib;` is preferred for the `meta` attribute, and since licenses rarely change, we don't need to use an array for that.

```suggestion
  meta = with lib; {
    homepage = "https://github.com/ikrivosheev/pa-notify";
    description = "PulseAudio or PipeWire volume notification";
    license = licenses.mit;
    maintainers = with maintainers; [ juancmuller ];
    mainProgram = "pa-notify";
    platforms = platforms.unix;
  };
```